src/Common/src/System/Security/Cryptography/CngPkcs8.cs

```diff
@@ -0,0 +1,316 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Security.Cryptography.Asn1;
+
+namespace System.Security.Cryptography
+{
+    internal static partial class CngPkcs8
+    {
+        // Windows 7, 8, and 8.1 don't support PBES2 export, so use
+        // the 3DES-192 scheme from PKCS12-PBE whenever deferring to the system.
+        private static readonly PbeParameters s_platformParameters =
+            new PbeParameters(
+                PbeEncryptionAlgorithm.TripleDes3KeyPkcs12,
+                HashAlgorithmName.SHA1,
+                10000);
```

Comment:
10000?

Reply:
That's a number that's small enough to be less than 0.1 seconds on almost all computers, but big enough to disassociate the derived key from the password.  But, since we still would have the password in memory (because we'd be about to decrypt/re-encrypt) we could probably reduce this down to 1, with something like this statement as justification.

One presumes that if Windows is not currently rejecting numbers that are "too small" that they will continue to not reject them.

Reply:
Makes sense, might be useful to put a comment saying something similar